notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-798_analysis.ipynb

```diff
@@ -0,0 +1,655 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.                        imsColumn = 'yPosition', 
</pre><p> </p><pre class="ql-syntax" spellcheck="false">NameError: name 'req_rms' is not defined
</pre><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/94/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Removed this cell (deprecated, it is a copy of the previous one with a different column but with an obsolete name). Actually only the first is necessary, and then you would change the IMS column name of interest.